python/lsst/obs/base/cameraMapper.py

```diff
@@ -460,7 +461,9 @@ def getVisitInfo(datasetType, pythonType, location, dataId):
 
                             setMethods("visitInfo", bypassImpl=getVisitInfo)
 
-                            # TODO: deprecate in DM-27170, remove in DM-27177
+                            # TODO: remove in DM-27177
+                            @deprecated(reason="Replaced with getFilterLabel. Will be removed after v23.",
```

Comment:
Should this be "after v22"? I thought (per the Community post) that DM-27177 would be as soon as that version comes out, and version 23 would use the "filter" name only for `FilterLabel`.

Reply:
Also, while our style guide is a bit inconsistent about this, I think `@deprecated` should have a `version` keyword.

Reply:
Version number question delegated to RFC-748.